### ISSUE TITLE
Running unit tests and integration tests

### ISSUE THREAD

**Issue by GoogleCodeExporter** [google/wicked-good-xpath#20]
```
Great library. How to run the tests?


```

Original issue reported on code.google.com by `mart...@enketo.org` on 11 Mar 2015 at 6:03
- [ ] Add a method to run the integration tests.
- [ ] Add a method to run the unit tests.

**Comment by Dominator008** [comment]
Internally at Google, the tests are run with Bazel using rules similar to those in https://github.com/bazelbuild/rules_closure. I will see if those rules can be used to run the tests in this project.

**Comment by ScottFreeCode** [comment]
Is the difference between the `test.js` filename and `test_js_deps.js` referenced in the pages significant?

**Comment by Dominator008** [comment]
@ScottFreeCode Yes they are different. `test.js` contains utilities for running the tests, while `test_js_deps.js` is an automatically generated file specifying what to `goog.require`. I guess you can comment out the line
```html
<script src="./test_js_deps.js"></script>
```
, see what's missing, and include them manually.

**Comment by ScottFreeCode** [comment]
Alright, finally got it working; thanks!

**Comment by Dominator008** [comment]
@ScottFreeCode Good to hear!

**Comment by Dominator008** [comment]
- [] Add a method to run the integration tests.
- [] Add a method to run the unit tests.

**Comment by bedney** [comment]
I'm still a bit confused here. So is there a way to run the tests automatically or do we have to run them one at a time?

**Comment by Dominator008** [comment]
@bedney I'm working on a workflow that will allow running tests out of the box.

**Comment by bedney** [comment]
@Dominator008 - Thank you sir! I appreciate it!
